templates/wafer/nav.html

```diff
@@ -0,0 +1,77 @@
+{% load i18n %}
+<div class="navbar navbar-inverse navbar-static-top">
+  <div class="container">
+    <div class="col-md-12">
+      <div class="navbar-header">
+        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#wafer-navbar-collapse">
+          <span class="sr-only">
+             Toggle navigation
+          </span>
+          <!-- To display a nav menu button on a small screen -->
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+        </button>
+        <a class="navbar-brand" href="{% url 'wafer_page' 'index' %}">{{ WAFER_CONFERENCE_NAME }}</a>
+      </div>
+      <div class="navbar-collapse collapse" id="wafer-navbar-collapse">
+        <ul class="nav navbar-nav">
+        {% for item in WAFER_MENUS.items %}
+          {% if item.image and item.url %}
+          <li class="icon">
+             <a href="{{ item.url }}" alt="{{ item.label }}">
+                   <img src="{{ item.image }}" />
+             </a>
+          </li>
+          {% elif item.url %}
+          <li><a href="{{ item.url }}">
+              {{ item.label }}
+          </a></li>
+          {% else %}
+          <li class="dropdown">
+            <a class="dropdown-toggle" data-toggle="dropdown" href="#">
+              {{ item.label }}<b class="caret"></b>
+            </a>
+            <ul class="dropdown-menu">
+              {% for subitem in item.items %}
+              <li><a href="{{ subitem.url }}">
+                  {{ subitem.label }}
+              </a></li>
+              {% endfor %}
+            </ul>
+          </li>
+          {% endif %}
+        {% endfor %}
+        </ul>
+        <ul class="nav navbar-nav navbar-right">
+          {% if user.is_authenticated %}
+          <li class="dropdown">
+            <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+              {{ user.username }}<b class="caret"></b>
+            </a>
+            <ul class="dropdown-menu">
+              <li><a href="{% url 'wafer_user_profile' username=user.username %}">
+                  {% blocktrans with name=user.userprofile.display_name %}
+                  {{ name }}'s profile
+                  {% endblocktrans %}
+              </a></li>
+              <li><a href="{% url 'auth_logout' %}">
+                  {% trans 'Log out' %}
+              </a></li>
+              {% if user.is_staff %}
+              <li><a href="{% url 'admin:index' %}">
+                  {% trans 'Site admin' %}
+              </a></li>
+              {% endif %}
+            </ul>
+          </li>
+          {% elif not WAFER_HIDE_LOGIN %}
+          <li><a href="{% url 'auth_login' %}">
+              {% trans 'Register / Log In' %}
+          </a></li>
+          {% endif %}
+        </ul>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
Yeah, and I think I extended it for what might be a Bootstrap 3 alignment issue that should be fixed in wafer too.